spring-cloud-azure-feature-management/src/main/java/com/microsoft/azure/spring/cloud/feature/manager/FeatureFilter.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.feature.manager;
+
+import org.springframework.stereotype.Component;
+
+import com.microsoft.azure.spring.cloud.feature.manager.entities.FeatureFilterEvaluationContext;
+
+/**
+ * A Filter for Feature Management that is attached to Features.
+ */
+@Component
```

Comment:
`@Component` for interface?

Reply:
Not sure what I was thinking here. The classes that implement it will need to be `@Components` I will just updated the class description.